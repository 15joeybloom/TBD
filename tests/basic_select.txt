select sname, sid, majorid from student
