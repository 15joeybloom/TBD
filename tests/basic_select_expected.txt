 majorid     sname   sid
------------------------
      10       joe     1
      20       amy     2
      10       max     3
      20       sue     4
      30       bob     5
      20       kim     6
      30       art     7
      20       pat     8
      10       lee     9
